doc/cookbook/service_proxy.md

```diff
@@ -115,7 +115,7 @@ syncInterval: 10s
 ```
 
 
-2. Create a pipeline and set its `serviceRegisty` field into `consul-001` and it will look up the consul configuration for the service named `springboot-application-order` as in filed `serviceName`.    
+2. Create a pipeline and set its `serviceRegisty` field into `eureka-001` and it will look up the eureka configuration for the service named `springboot-application-order` as in filed `serviceName`.    
```

Comment:
there are other typo:

```suggestion
2. Create a pipeline and set its `serviceRegistry` field into `eureka-001` and it will look up the eureka configuration for the service named `springboot-application-order` as in field `serviceName`.    
```

Reply:
Hi, @zhangfh517 , the `serviceRegisrty` in example YAML also need to be corrected into `serviceRegistry`, can u fix them in this PR too, thx.

Reply:
@benja-wu fixed